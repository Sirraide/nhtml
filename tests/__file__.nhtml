h1

eval { $('h1').text = __file__ }