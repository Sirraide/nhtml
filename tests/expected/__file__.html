<h1>__file__.nhtml</h1>
